File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_ControlTypes.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_ControlTypes } from './Sys_ControlTypes.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_ControlTypesService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ControlType_Id', 'GroupCode', 'ControlType_Code', 'ControlType_Name', 'Description'];

    List: Sys_ControlTypes[] | undefined;
    ListFilter: Sys_ControlTypes[] | undefined;
    FormData!: Sys_ControlTypes;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_ControlTypes";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_ControlTypesService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_ControlTypesService : BaseService<Sys_ControlTypes, ISys_ControlTypesRepository>
    , ISys_ControlTypesService
    {
    private readonly ISys_ControlTypesRepository _Sys_ControlTypesRepository;
    public Sys_ControlTypesService(ISys_ControlTypesRepository Sys_ControlTypesRepository) : base(Sys_ControlTypesRepository)
    {
    _Sys_ControlTypesRepository = Sys_ControlTypesRepository;
    }
    }
    }

